pkg/ccl/backupccl/show.go

```diff
@@ -721,13 +729,21 @@ func backupShowerDefault(
 					dataSizeDatum := tree.DNull
 					rowCountDatum := tree.DNull
 					fileSizeDatum := tree.DNull
+					regionsDatum := tree.DNull
 
 					desc := descbuilder.NewBuilder(descriptor).BuildExistingMutable()
 
 					descriptorName := desc.GetName()
 					switch desc := desc.(type) {
 					case catalog.DatabaseDescriptor:
 						descriptorType = "database"
+						if desc.IsMultiRegion() {
+							regions, err := showRegions(typeIDToTypeDescriptor[desc.GetRegionConfig().RegionEnumID], desc.GetName())
+							if err != nil {
+								log.Errorf(ctx, "error while generating regions column data: %+v", err)
```

Comment:
why do you think this error should only get logged? In general, an error should cause the sql cmd to fail and return immediately to the user.

also, i'd rewrite the error message as: `errors.Wrapf("cannot generate regions column",err)`

There's no need to include the word "error" in the error msg, as all error messages will be prefixed with "error:"